docs/content/topics/operators/security.md

```diff
@@ -51,30 +58,31 @@ secrets, and persistent volume claims.
 
 ## Project Security
 
-Brigade is opinionated about configuring projects and storing data like credentials.
-Because sensitive information is stored in a project's secret, care should be
-taken in preventing unauthorized access to that secret.
+Brigade is opinionated about configuring projects and storing data like
+credentials. Sensitive information relevant to a project should be set as
+[Secrets] on that project. In turn, project secrets are stored in a Kubernetes
+secret, so care should be taken in preventing unauthorized access to these
+resources.
 
-- Out-of-the-box, the project is the location where credentials should be stored
+- Out-of-the-box, credentials should be stored as project secrets
 - A project's credentials are accessible to any script running in that project,
   regardless of event.
-- For SSH-based Git clones, the SSH key should be stored on the project.
+- For SSH-based Git clones, the SSH key should be stored as a project secret.
```

Comment:
iirc, there's a well-known secret name used for this purpose. Should we document that here?